BUILD_LINUX.md

```diff
@@ -150,7 +150,7 @@ Go to localhost in the running interface.
 
 #### Notes
 
-If your goal is to setup a development environment, it is desirable to set the
+If your goal is to setup a development environment, it is desirable to set up the
```

Comment:
setup -> set up

Reply:
Fixed.

Reply:
Actually I'm confused. :)

Reply:
```
If your goal is to setup a development environment, …
``
should be
```
If your goal is to set up a development environment,
```

The extra "up" you subsequently added should be removed.

Reply:
Okay, fixed.